Sources/TuistKit/Utils/Sequence+KeyPath.swift

```diff
@@ -0,0 +1,23 @@
+extension Sequence {
```

Comment:
```swift
import Foundation
```

Reply:
Technically these don't need Foundation as they only use code in the Stdlib